jetty-http2/http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Session.java

```diff
@@ -1239,7 +1257,8 @@ public void succeeded()
     private class DataEntry extends HTTP2Flusher.Entry
     {
         private int bytes;
-        private int dataRemaining;
+        private int frameBytes;
+        private int dataBytes;
```

Comment:
frameBytes and dataBytes are not good names.  Is this in total? a progressive value?  So a better name or a comment